src/screens/FeedScreen/hooks/useCoreFeed.js

```diff
@@ -69,11 +69,11 @@ const useCoreFeed = () => {
       };
       return color;
     }
-    const randomIndex = getRandomInt(0, listFeedColor.length);
+    const randomIndex = getRandomInt(0, listFeedColor.length - 1);
     let newColor = listFeedColor[randomIndex];
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Good job on adjusting the range of `getRandomInt` to prevent potential out-of-bounds access. This change ensures that the random index will always be within the valid range of the array.



<!-- This is an auto-generated comment by OSS CodeRabbit -->